packages/bpk-component-badge/src/BpkBadge-test.js

```diff
@@ -47,4 +47,39 @@ describe('BpkBadge', () => {
       .toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  it('should render correctly when success', () => {
```

Comment:
We could reduce the code here a fair bit by iterating over `BADGE_TYPES` and testing it that way.

See here for an example: https://github.com/Skyscanner/backpack/blob/master/native/packages/react-native-bpk-component-button/src/BpkButton-test.common.js#L105